roles/confluent.kafka_broker/tasks/main.yml

```diff
@@ -231,6 +231,7 @@
   loop: "{{ sasl_scram_users|dict2items }}"
   run_once: true
   when: "'SCRAM-SHA-512' in kafka_broker_sasl_enabled_mechanisms"
+  no_log: true
```

Comment:
Can you put in the description what the log looks like now? I'm curious the before/after

Reply:
done!